includes/definitions/lenovo-cnos.yaml

```diff
@@ -0,0 +1,12 @@
+os: lenovo-cnos
```

Comment:
You can rename this to `cnos` as that's the OS name. It doesn't need the vendor before it unless we have duplicates.

You'll need to rename the files naturally.